frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -163,6 +197,29 @@ class MeshesView extends React.Component<Props, State> {
       Store.dispatch(setPositionAction(seedPosition, null, false));
     };
 
+    const startComputingMeshfile = async () => {
+      await startComputeMeshFileJob(
+        this.props.organization,
+        this.props.datasetName,
+        this.props.segmentationLayer.fallbackLayer || this.props.segmentationLayer.name,
+        this.props.activeResolution,
+      );
+      this.setState({ isComputingMeshfile: true });
+      Toast.success(
+        "The computation of a mesh file was started successfully. Depending on the dataset's size this may take a bit. You don't need to keep this dataset open for the computation to continue.",
+      );
+    };
+
+    const getComputeMeshFileButton = () => (
+      <Button
+        size="small"
+        loading={this.state.isComputingMeshfile}
+        onClick={startComputingMeshfile}
+      >
```

Comment:
I think, this button will also appear (and be clickable) if the wk instance does not support jobs. I suggest to use `features().jobsEnabled` to make this button potentially `disabled`. In that case, the button should display a tooltip on hover which says something like this:

`The computation of mesh files is not supported by this webKnossos instance.`